packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCard.tsx

```diff
@@ -23,8 +24,14 @@ export class DocumentCard extends React.Component<IDocumentCardProps, any> {
       };
     }
 
+    // if this element is actionable it should have an aria role
+    let role = actionable ? (onClick ? 'button' : 'link') : null;
+    let tabIndex = actionable ? 0 : null;
+
     return (
       <div
+        tabIndex={ tabIndex }
```

Comment:
Never mind, I think focusZone will make sure set the unactive child element tabindex to -1 even it was 0. So my scenario won't be break. The way I achieved the accessibility is that  I created a wrap component to wrap document card inside. And I add key board handling at that level. And use the wrapped component in a list. 

<div className='ms-DocumentCardTile' data-is-focusable={ true } tabIndex={ 0 } onKeyDown={ this._onKeyDown } role='gridcell'
        aria-label={ ariaLabel } aria-describedby={ ariaDescribedByElementId }>
        <FocusZone
          direction={ FocusZoneDirection.vertical }>
          { useCompactDocumentCard ? this._renderCompactDocumentCard() : this._renderNormalDocumentCard() }
        </FocusZone>
      </div>